Documentation/INTERNALS.md

```diff
@@ -226,9 +249,8 @@ message Foo {
 }
 ```
 
-This generates a storage class, of course. When the serializer invokes
-`traverse()` on the struct, it simply passes the visitor to `traverse()` on the
-storage class, which looks like this:
+This generates a storage class, of course.
+The storage class and the generated `traverse()` looks like this:
```

Comment:
Is it worth pointing out the traverse goes in field order (hence the complications with extensions and oneofs)?